GOVERNANCE.md

```diff
@@ -0,0 +1,62 @@
+
+### Contributors
+Contributors are community members who either have no desire to become committers, or have not yet been given the opportunity by the benevolent dictator. They make valuable contributions, such as those outlined in the list below, but generally do not have the authority to make direct changes to the project code. Contributors engage with the project through communication tools, such as email lists, and via reports and patches attached to issues in the issue tracker.
+
+Anyone can become a contributor. There is no expectation of commitment to the project, no specific skill requirements and no selection process. To become a contributor, a community member simply has to perform one or more actions that are beneficial to the project.
+
+Some contributors will already be engaging with the project as users, but will also find themselves doing one or more of the following:
+
+*   supporting new users (current users often provide the most effective new user support)
+*   reporting bugs
+*   identifying requirements
+*   supplying graphics and web design
+*   programming
+*   assisting with project infrastructure
+*   writing documentation
+*   fixing bugs
+*   adding features
+
+As contributors gain experience and familiarity with the project, they may find that the project lead starts relying on them more and more. When this begins to happen, they gradually adopt the role of committer, as described above.
+
+### Users
+Users are community members who have a need for the project. They are the most important members of the community: without them, the project would have no purpose. Anyone can be a user; there are no specific requirements.
+
+Users should be encouraged to participate in the life of the project and the community as much as possible. User contributions enable the project team to ensure that they are satisfying the needs of those users. Common user activities include (but are not limited to):
+
+*   evangelising about the project
+*   informing developers of project strengths and weaknesses from a new user’s perspective
+*   providing moral support (a ‘thank you’ goes a long way)
+*   providing financial support
+
+Users who continue to engage with the project and its community will often find themselves becoming more and more involved. Such users may then go on to become contributors, as described above.
+
+## Support
```

Comment:
Somewhere in this section, should we link to our support page on the website?
https://forsetisecurity.org/docs/latest/use/get-help.html

Reply:
done